index.js

```diff
@@ -13,7 +13,11 @@ module.exports = async (url, timeout = 3000) => {
 	const getUrlData = new Promise((resolve, reject) => {
 		const req = request.get(url, {timeout}, (err, response, html) => {
 			if (err) {
-				reject(err);
+				return reject(err);
```

Comment:
What impact does this change make?

Reply:
Stops the rest of the script from processing